src/v0/destinations/zendesk/transform.ts

```diff
@@ -506,29 +493,22 @@ async function createOrganization(
   const url = baseEndpoint + category.createEndpoint;
   const config = { headers };
 
-  try {
-    const resp = await myAxios.post(url, payload, config, {
-      destType: 'zendesk',
-      feature: 'transformation',
-      endpointPath: '/organizations/create_or_update.json',
-      requestMethod: 'POST',
-      module: 'router',
-      metadata,
-    });
-
-    if (!resp.data || !resp.data.organization) {
-      logger.debug(`${NAME}:: Couldn't create Organization: ${message.traits.name}`);
-      return undefined;
-    }
+  const { processedResponse: resp } = await handleHttpRequest('post', url, payload, config, {
+    destType: 'zendesk',
+    feature: 'transformation',
+    endpointPath: '/organizations/create_or_update.json',
+    requestMethod: 'POST',
+    module: 'router',
+    metadata,
+  });
 
-    const orgId = resp?.data?.organization?.id;
-    return orgId;
-  } catch (error) {
-    logger.debug(
-      `${NAME}:: Couldn't create Organization: ${message.traits.name} and error: ${error}`,
-    );
+  if (!isHttpStatusSuccess(resp.status) || !resp.response || !resp.response.organization) {
```

Comment:
[Nitpick] Can be simplified.
```suggestion
  if (!isHttpStatusSuccess(resp.status) || !resp.response?.organization) {
```